lib/src/chart/pie_chart/pie_chart_data.dart

```diff
@@ -192,6 +194,9 @@ class PieChartSectionData with EquatableMixin {
   /// Defines the radius of section.
   final double radius;
 
+  /// Defines the border radius of section.
+  final double borderRadius;
```

Comment:
You also forgot to add this new property in the `List<Object?> get props => []` getter